src/components/document/DocumentView/DocumentView.jsx

```diff
@@ -1,3 +1,17 @@
-const DocumentView = () => { return <div>DocumentView</div> }
+import TextEditor from '@/components/TextEditor.jsx';
+import docText from "@/assets/example_document";
```

Comment:
the doc gets passed as a prop `document` anyways, shouldn't need to import this directly (or import it directly period, we should be using the doc api)